requirements.txt

```diff
@@ -13,39 +13,41 @@ mypy-extensions==1.0.0; python_version >= '3.5'
 packaging==23.2; python_version >= '3.7'
 paramiko==3.4.0; python_version >= '3.6'
 pathspec==0.12.1; python_version >= '3.8'
-phoenix6==24.0.0b5; python_version >= '3.7' and python_version < '4'
```

Comment:
We should remove the requirements.txt file but I can do that in the branch I was playing around with fixing up the github workflows